definitions/private-law/json/State.json

```diff
@@ -109,5 +109,14 @@
     "Description": "Requested for deletion",
     "DisplayOrder": 12,
     "TitleDisplay": "# #${[CASE_REFERENCE]}\n ## Case Name: ${applicantCaseName}\n ## Family Man ID: ${familymanCaseNumber}"
+  },
+  {
+    "LiveFrom": "26/04/2023",
+    "CaseTypeID": "PRLAPPS",
+    "ID": "PROCEEDS_IN_HERITAGE_SYSTEM",
+    "Name": "Proceed in heritage system",
+    "Description": "Proceed in heritage system",
```

Comment:
Can we change this description to say 'Proceeding in offline mode in familyman system'

Reply:
Changed thr description to 'Proceeding in offline mode in familyman system'